pkgs/development/python-modules/gradio/default.nix

```
@@ -0,0 +1,140 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, writeTextFile
+, setuptools
+, analytics-python
+, aiohttp
+, fastapi
+, ffmpy
+, markdown-it-py#[linkify,plugins]
```

```suggestion
, markdown-it-py
```

Please use optional-dependencies